installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_gcp_rules.json

```diff
@@ -2023,7 +2023,7 @@
   {
     "policyId": "GCP_Cloud_Function_configured_with_overly_permissive_Ingress_setting",
     "policyUUID": "GCP_Cloud_Function_configured_with_overly_permissive_Ingress_setting",
-    "policyName": "GCP_Cloud_Function_configured_with_overly_permissive_Ingress_setting",
+    "policyName": "Check permission of Ingress Setting of GCP cloud Function",
```

Comment:
Need to reduce the policyUUid and policyId

Reply:
Please check now